index.html

```diff
@@ -11,54 +11,153 @@
+                            <div>Back End Dev</div>
+                            <div class="dot"></div>
+                            <div>2015</div>
+                        </div>
+                    </div>
+                    <div class="content">A daily selection of privately personalized reads; no accounts or sign-ups required.</div>
+                    <div class="categories">
+                        <ul>
+                            <li>html</li>
+                            <li>css</li>
+                            <li>javaScript</li>
+                        </ul>
+                    </div>
+                    <button type="button">See Project</button>
+                </div>
                 
-                <ul>
-                    <li><a href="https://twitter.com"><i class="fab fa-twitter"></i></a></li>
-                    <li><a href="https://facebook.com"><i class="fab fa-facebook"></i></a></li>
-                    <li><a href="https://linkedin.com"><i class="fab fa-linkedin-in"></i></a></li>
-                    <li><a href="https://github.com"><i class="fab fa-github"></i></a></li>
-                    <li><a href="https://youtube.com"><i class="fab fa-youtube"></i></a></li>
-                </ul>
+            
             </section>
-        </main>
-    </div>
-</body>
 
+            <section class="about">
+                <div class="headline">
+                    About<br>Myself
+                </div>
 
+                <div class="body">
+                    <p>
+                        I’m a software developer! I can help you build a product feature or website. Look through some
+                        of my work and experience! If you like what you see and have a project you need coded, don’t
+                        hesitate to contact me.
+                    </p>
+                </div>
 
+                <div class="social-media">
+                    <p>LET'S CONNECT</p>
+                    <ul>
+                        <li><a href="https://twitter.com"><i class="fab fa-twitter"></i></a></li>
+                        <li><a href="https://facebook.com"><i class="fab fa-facebook"></i></a></li>
+                        <li><a href="https://linkedin.com"><i class="fab fa-linkedin-in"></i></a></li>
+                        <li><a href="https://github.com"><i class="fab fa-github"></i></a></li>
+                        <li><a href="https://youtube.com"><i class="fab fa-youtube"></i></a></li>
+                    </ul>
+                </div>
 
+                <div><button type="button">Get my resume</button></div>
 
+                <div class="about-skills">
+                    <ul>
+                        <li><img src="/images/Frame12.svg" alt=""></li>
+                        <li><img src="/images/Frame8.svg" alt=""></li>
+                        <li><img src="/images/Frame9.svg" alt=""></li>
+                    </ul>
+                </div>
+            </section>
```

Comment:
- [ ] According to the requirements, You must stick to the design as much as possible (e.g. font, colors, images, tests, margins). So Kindly, change the background color in the about section to look similar to the [Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122)
> It should be white `background-color: #fff` instead of `background-color: #e5e5e5`

<img src="https://user-images.githubusercontent.com/45231276/176984712-c63783da-232b-4a90-9974-0363d6836306.PNG" width="500" />